src/main/java/org/foundation/coding/movie/NewReleaseMovie.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Refactor Code Tasks.
+ */
+package org.foundation.coding.movie;
+
+/**
+ * Type of movie Release.
+ */
+public class NewReleaseMovie extends Movie {
+  //Code of release.
+  public static final int NEW_RELEASE = 1;
+  public static final int PRECIO_BY_DEFAULT = 3;
+  /**
+   * constructor.
+   * @param title receive.
+   * @param priceCode receive.
+   */
+  public NewReleaseMovie(final String title, final int priceCode) {
+    super(title, priceCode);
+  }
+
+  /**
+   * calculate the fee.
+   * @param days receive.
+   * @return fee.
+   */
+  @Override
+  public double getFee(int days) {
+
+    return days * PRECIO_BY_DEFAULT;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)